1.3.0/create.py

```diff
@@ -0,0 +1,108 @@
+"""
+Add age.[train|dev|test] tables with speaker information
+consists of randomly selected 20 emotionally neutral samples per speaker
+all tables being age/gender balanced
+"""
+
+import random
+import pandas as pd
+
+import audb
+import audeer
+import audformat
+import util
+import trainDevTestSplit
+
+# make it reproducible
+random.seed(23)
+
+image_dir = "images/"
+
+
+def main():
+    name = "crema-d"
+    previous_version = "1.2.0"
+
+    build_dir = "../build"
```

Comment:
Since we now can use `only_metadata=True` it is preferred to no longer have a shared build folder between versions:

```suggestion
    build_dir = "./build"
```